app/views/tutorial/a9.html

```diff
@@ -19,7 +19,17 @@ <h3 class="panel-title">Description</h3>
             <div class="panel-body">
                 <p>Components, such as libraries, frameworks, and other software modules, almost always run with full privileges. If a vulnerable component is exploited, such an attack can facilitate serious data loss or server takeover. Applications using components with known vulnerabilities may undermine application defenses and enable a range of possible attacks and impacts.
                 </p>
-                <p>Using insecure npm packages can lead to this vulnerability. The <a href="https://nodesecurity.io/advisories">Node Security project </a> is a great initiative and resource to know about related vulnerabilities.</p>
+                <p>Using insecure npm packages can lead to this vulnerability. Some projects today help test and alert on insecure dependencies:
+                    <ol>
+                        <li>
+                            The <a href="https://nodesecurity.io/advisories">Node Security project </a> is a great initiative and resource to know about related vulnerabilities.
+                        </li>
+                        <li>
+                            <a href="https://snyk.io/">Snyk.io </a> is another Node.js CLI tool and Platform to scan and detect vulnerable packages
```

Comment:
Also bithound, npm-outdated https://leanpub.com/holistic-infosec-for-web-developers-fascicle1-vps-network-cloud-webapplications/read#web-applications-countermeasures-consuming-free-and-open-source-tooling